stdlib/public/Concurrency/TaskGroup.swift

```diff
@@ -33,52 +33,45 @@ extension Task {
 }
 
 
-/// Starts a new task group which provides a scope in which a dynamic number of
-/// tasks may be spawned.
+/// Starts a new scope in which a dynamic number of tasks can be spawned.
 ///
-/// Tasks added to the group by `group.spawn()` will automatically be awaited on
-/// when the scope exits. If the group exits by throwing, all added tasks will
-/// be cancelled and their results discarded.
+/// When the group returns,
+/// it implicitly waits for all spawned tasks to complete.
+/// The tasks are canceled only if `cancelAll()` was invoked before returning,
+/// if the groups' task was canceled,
+/// or if the group body throws an error.
+/// ◊TR: Is it possible to throw an error here?
```

Comment:
so there is `withThrowingTaskGroup` and `withTaskGroup` - throwing is only possible in the throwing one; perhaps the comment is currently copied (wrongly) to both?